addons/account/models/mail_scheduled_message.py

```diff
@@ -0,0 +1,111 @@
+from odoo import api, fields, models, _
+from odoo.fields import Datetime
+
+
+class MailScheduledMessage(models.Model):
+    _inherit = 'mail.scheduled.message'
+
+    # JSON field to store all move related data to generate and send invoice
+    scheduled_send_payload = fields.Json()
```

Comment:
This should be avoided, as it makes no sense for other models, and globally is a "garbage" field: generic name, no clear explanation, ... you may use ``notification_parameters``, in combination with clear ``_get_notify_valid_parameters`` override, something like that, to be more inlined with classic mail. I know account module has a long history of custom composer for a lot of reasons, but if you want to use standard mail tools (which is probably a good idea), better align with their usage :) .